server/src/test/java/com/google/coffeehouse/common/PersonTest.java

```diff
@@ -36,68 +41,101 @@
   private static final String ALT_PRONOUNS = "New Pronouns";
   private static final String IDENTIFICATION_STRING = "predetermined-identification-string";
   private Person.Builder personBuilder;
+  private Map personInfo;
   
   @Mock private IdentifierGenerator idGen;
 
   @Before
   public void setUp() {
-    idGen = Mockito.mock(IdentifierGenerator.class);
-    Mockito.when(idGen.generateId()).thenReturn(IDENTIFICATION_STRING);
+    personInfo = new HashMap<String, String>();
+
+    idGen = mock(IdentifierGenerator.class);
+    when(idGen.generateId()).thenReturn(IDENTIFICATION_STRING);
 
     personBuilder = Person.newBuilder(EMAIL, NICKNAME);
   }
 
   @Test
-  public void getExistingNickname() {
+  public void getNickname_exists() {
     Person p = personBuilder.build();
-    Assert.assertEquals(NICKNAME, p.getNickname());
+    assertEquals(NICKNAME, p.getNickname());
```

Comment:
Is there any specific reason as to why you changed the import to do assertEquals instead? Just curious so I can change my tests accordingly.

Reply:
Keaton mentioned statically importing classes that you only call static methods on. It's certainly personal preference but the argument is that it makes the code cleaner and easier to write. We should talk about this as a team though!

Reply:
That makes sense, I agree!